elixir/apps/web/test/web/live/groups/edit_actors_test.exs

```diff
@@ -113,7 +113,6 @@ defmodule Web.Live.Groups.EditActorsTest do
     end)
     |> with_table_row("actor", "#{admin_actor.name} (admin)", fn row ->
       for(identity <- admin_actor.identities) do
-        assert row["identities"] =~ identity.provider.name
```

Comment:
The icon is not pulled by the `with_table_row` helper. The provider_icon matcher is tested elsewhere so I thought this was ok to remove.